### DISCUSSION TITLE
Linux support

### DISCUSSION THREAD

**Discussion by sagashiman** [syt0r/Kanji-Dojo#134] [General]
I wonder if there any chance that there are plans to create desktop client, particularly for Linux. 

I think for some people - including me - learning process is more convenient with PC.

In my opinion, it would be more natural to learn kanji by drowning them with stilus connected to pc.

**Comment by syt0r** [comment]
Check the downloads section in readme and let me know if it works for you

**Comment by sagashiman** [comment]
I was able to run the app and practice some kanji with my stylus.  Great feeling. Thank you for that. 
Though, it is difficult to consider it perfect solution. There is no way to synchronize progress with my mobile device and  I need  to run a code in a terminal every time I need to run the app (perhaps because of not perfectly installed Java).

I understand that creating a native app for different platform is by no means a trivial task, but I hope some day it would be realized.
